content/accessibility/tools.mdx

```diff
@@ -58,3 +61,12 @@ Free application for Windows that is comparable to Sim Daltonism on Mac.
 Color review is fantastic if you're looking for help finding an accessible color in an existing palette. 
 
 ![""](https://user-images.githubusercontent.com/40274682/88444756-90d6a100-cdd3-11ea-876c-1c850ae19223.png)
+
+### [WebAIM Contrast Checker](https://webaim.org/resources/contrastchecker/)
+
+The WebAIM contrast checker is the golden standard and does a good job explaining what the result means. 
+At GitHub the WebAIM contrast checker is the standard that has to be met.
+
+There is also a [link contrast checker](https://webaim.org/resources/linkcontrastchecker/?fcolor=D9D9D9&bcolor=FFFFFF).
+
+!["webaim contrast color checker website"](https://user-images.githubusercontent.com/813754/188844756-2340bcbd-2630-4a5f-be4a-8796774601cc.png)
```

Comment:
If this is the standard we should hit, let's put it before the color review website 🙂.

Reply:
Otherwise, all looks good!

Reply:
Will do.